.ci/jobs.t/elastic+elasticsearch+pull-request+build-benchmark-part1.yml

```diff
@@ -46,4 +46,5 @@
          $WORKSPACE/.ci/scripts/run-gradle.sh :build-tools-internal:bootstrapPerformanceTests
          $WORKSPACE/.ci/scripts/install-gradle-profiler.sh
          $WORKSPACE/.ci/scripts/run-gradle-profiler.sh --benchmark --scenario-file build-tools-internal/build/performanceTests/elasticsearch-build-benchmark-part1.scenarios --project-dir . --output-dir profile-out
-         tar -czf build/${BUILD_NUMBER}.tar.bz2 profile-out
\ No newline at end of file
+         mkdir $WORKSPACE/build
```

Comment:
The PR description reads like the `build` may or may not be there. If it already exists, then the `mkdir` call will fail. If so, we should call `mkdir -p` instead.